datahub-web-react/src/app/entity/dataset/profile/schema/components/SchemaDescriptionField.tsx

```diff
@@ -101,9 +101,9 @@ export default function DescriptionField({
             await onUpdate(desc || '');
             // message.destroy();
             // message.success({ content: 'Updated!', duration: 2 });
-        } catch (e) {
+        } catch (e: unknown) {
```

Comment:
Are there cases where "e" is not an Error? I have not encountered them but interested to hear if you've seen this!

Reply:
I think this is the [original proposal](https://github.com/microsoft/TypeScript/issues/36775).  It tries to be more specific on the error types, like other languages.  Like the example from that thread
```
try {
  externalApi()
} catch(e) {
  if (e instanceof Error) {
    // Probably send this error to developer of API
  } else if (e instanceof ApiError) {
    // We messed up, log it and fix it in next patch
  }
}
```